Games/types/Mafia/roles/cards/IssueChallenge.js

```diff
@@ -0,0 +1,91 @@
+const Card = require("../../Card");
+const { PRIORITY_CHALLENGE, PRIORITY_DAY_DEFAULT } = require("../../const/Priority");
+
+module.exports = class IssueChallenge extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Jail Target": {
+                states: ["Day"],
+                flags: ["voting"],
+                action: {
+                    labels: ["challenge"],
+                    priority: PRIORITY_DAY_DEFAULT,
+                    run: function () {
+                        if (this.dominates()) {
+                            this.target.holdItem("Challenge");
+                            this.actor.holdItem("Challenge");
+                            this.data.chalenged = this.target;
```

Comment:
I feel that it's better to use challengeTarget, this word has negative connotations

Alternatively, just call them Gambler (actor) vs Challenger (target)